common/views/components/DescriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,54 @@
+import { FunctionComponent, ReactElement } from 'react';
+import styled from 'styled-components';
+import { classNames, font } from '@weco/common/utils/classnames';
+
+const List = styled.dl`
+  display: grid;
+  grid-column-gap: 20px;
+  grid-row-gap: 6px;
+  grid-template-columns: min-content 1fr;
+  margin: 0;
+`;
+
+const Term = styled.dt.attrs({
+  className: classNames({
+    [font('hnb', 5)]: true,
+  }),
+})``;
+
+const Description = styled.dd.attrs({
+  className: classNames({
+    [font('hnr', 5)]: true,
+  }),
+})`
+  margin-left: 0;
+`;
+
+type Props = {
+  title: string;
+  items: { term: string; description: string | ReactElement }[];
```

Comment:
Why do we need to support both?

Reply:
The access method `dd` _can_ contain a `ButtonLink` if the item is requestable online.